modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItTablesApiTest.java

```diff
@@ -347,8 +349,7 @@ public void testAddColumnFromLaggedNode() throws Exception {
     }
 
     /**
-     * Checks that if a table would be created/dropped into any cluster node, this is visible to all
-     * other nodes.
+     * Checks that if a table would be created/dropped into any cluster node, this is visible to all other nodes.
```

Comment:
"into any cluster node" - can`t understand it.

Reply:
This comment was changed. I hope, it becomes mo clear.